DEVELOPMENT.md

```diff
@@ -0,0 +1,106 @@
+# Developing
+
+## Getting started
+
+Many of the instructions here replicate what's in the [tektoncd/pipeline development guide](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md), with a couple of caveats currently.
+
+1. This project currently does not support being built with `ko`
+2. This project has not yet been tested with GKE
+3. The instructions here pertain to building the backend that lets us interact with Tekton resources. Frontend instructions are to follow.
+
+We would love to accomplish these tasks and to update this document, contributions welcome!
+
+### Requirements
+
+You must install these tools:
+
+1. [`go`](https://golang.org/doc/install): The language Tekton Dashboard is
+   built in
+1. [`git`](https://help.github.com/articles/set-up-git/): For source control
+1. [`dep`](https://github.com/golang/dep): For managing external Go
+   dependencies. - Please Install dep v0.5.0 or greater.
+1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
+   interacting with your kube cluster
+   
+### Checkout your fork
+
+The Go tools require that you clone the repository to the
+`src/github.com/tektoncd/dashboard` directory in your
+[`GOPATH`](https://github.com/golang/go/wiki/SettingGOPATH).
+
+To check out this repository:
+
+1. Create your own
+   [fork of this repo](https://help.github.com/articles/fork-a-repo/)
+1. Clone it to your machine:
+
+```shell
+mkdir -p ${GOPATH}/src/github.com/tektoncd
+cd ${GOPATH}/src/github.com/tektoncd
+git clone git@github.com:${YOUR_GITHUB_USERNAME}/dashboard.git
+cd dashboard
+git remote add upstream git@github.com:tektoncd/dashboard.git
+git remote set-url --push upstream no_push
+```
+
+_Adding the `upstream` remote sets you up nicely for regularly
+[syncing your fork](https://help.github.com/articles/syncing-a-fork/)._
+
+## Environment Setup
+
+We recommend Docker building, pushing, and replacing the yaml in `install` to refer to your built and pushed image.
```

Comment:
This sentence is confusing to me - replacing the yaml? maybe specify here which field in the yaml to alter